### PR TITLE
Initial config update section

### PR DESCRIPTION
**Pull request by hamishwillee**
@bkueng The initial config section was linked to as "an overview of the system" (which it isn't) and contained pretty much instructions to set up a standard config for your airframe using QGC daily build.

I updated this so that now it is "Tools you need to work as a developer on PX4" and (as before) "Use QGC to config your airframe in a standard way" - but with links to the user guide. 

It is definitely "more sane" but could use your advice/checking. For example, should we perhaps rename the doc in some way. Comments in line.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 This guide describes how to work inside the PX4 system architecture. It enables developers to:
 
-* Get an [overview of the system](en/setup/config_initial.md)
-* Access and modify the [PX4 Flight Stack](en/concept/flight_stack.md) and [PX4 Middleware](en/concept/middleware.md)
+* Get a [minimum developer setup](en/setup/config_initial.md).
```
Comment (hamishwillee) [inline README.md:7]: FYI, explain what it actually does. There doesn't seem to be a good "overall architecture" doc, so I suspect that is covered by "concepts". 

@bkueng Do you think we should have a bullet "understand the main PX4 concepts" pointing to that section?

File: en/setup/config_initial.md [hunk 2]
```diff
@@ -1,22 +1,34 @@
...
+  * Samsung Note 4 or equivalent / more recent Android tablet
+  * iPad (requires Wifi telemetry adapter)
+* Safety glasses
+* For multicopters - tether for more risky tests
+
+## Vehicle Configuration
+
+Download the [QGroundControl Daily Build](https://docs.qgroundcontrol.com/en/releases/daily_builds.html) for your development platform.
 
-## Radio Control Options
```
Comment (hamishwillee) [inline en/setup/config_initial.md]: @bkueng I deleted all this stuff. The relevant configuration is all in the PX4 user guide as linked

File: README.md (hunk 1 above)
Comment (bkueng) [reply README.md:7]: I agree the "overall architecture" page is still missing, so I'd leave it as is for now.

File: en/setup/config_initial.md [hunk 3]
```diff
@@ -1,22 +1,34 @@
...
-{% youtube %}https://www.youtube.com/watch?v=91VGmdSlbo4&rel=0&vq=hd720{% endyoutube %}
+The equipment below is highly recommended:
+* A Taranis Plus remote control for the safety pilot (or equivalent)
+* A development computer
+  * MacBook Pro or Air with OSX 10.10 or newer
+  * Modern laptop with Ubuntu Linux (14.04 or newer)
+* A ground control station device
+  * Any MacBook or Ubuntu Linux laptop (can be the development computer)
+  * Samsung Note 4 or equivalent / more recent Android tablet
+  * iPad (requires Wifi telemetry adapter)
```
Comment (bkueng) [inline en/setup/config_initial.md]: You can even add a smartphone. Most of them are so large nowadays that you can use QGC.

**Review by bkueng — APPROVED** [review]
This looks good.

File: README.md (hunk 1 above)
Comment (hamishwillee) [reply README.md:7]: OK, thank you. I'll leave that for now.

File: en/setup/config_initial.md (hunk 3 above)
Comment (hamishwillee) [reply en/setup/config_initial.md]: Thanks. Done!

**Comment by hamishwillee** [comment]
Updated following comments and merged. Thanks @bkueng .

**Pull request merged**